### PR TITLE
Start create SEV file

### PR DESCRIPTION
**Pull request by pm700**
description create SEV file

### Description of Change

SEV visit FR
### Test results

- Did you do any localized testing?

Never tested

Who should know about the change?
team

/cc: @githubteacher

### CODE REVIEW

File: sev-testfile.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+test SEV 1
```
Comment (pm700) [inline sev-testfile.md:1]: sev review

**Review by pm700 — COMMENTED** [review]
review OK

File: sev-testfile.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+test SEV 2
+
+test SEV 3
```
Comment (githubteacher) [inline sev-testfile.md:5]: Looks good! 👍

File: sev-testfile.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+test SEV 1
+
+test SEV 2
```
Comment (hd2605) [inline sev-testfile.md:3]: seems ok

**Review by hd2605 — APPROVED** [review]

**Review by githubteacher — APPROVED** [review]
This looks good!

File: sev-testfile.md (hunk 3 above)
Comment (Claire-CConti) [reply sev-testfile.md:3]: Ok for me too

**Pull request merged**